### PR TITLE
Update dAPI reference and guides section

### PR DESCRIPTION
**Pull request by vanshwassan**
Updated the `/guides` and `/reference` section for dAPIs for the upcoming market updates.
- Removed references regarding Self-funded and Managed dAPIs
- Updated the guides and reference sections for the new market updates and OEV
- Removed the Gelato guide for funding dAPI sponsor wallets

### CODE REVIEW

**Review by Ashar2shahid — APPROVED** [review]
Did some changes @dcroote can you take a look before I merge

File: docs/guides/dapis/subscribing-to-dapis/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+where you can find more information about the dAPI.
+
+<img src="../../assets/images/dapi-3.png" style="width:100%"/>
+
+If a dAPI is already activated, it will provide information like the current
+aggregated on-chain value, funding address, current configuration, and the
+expiration date.
+
+Once you have selected your dAPI, you will then be presented with an option to
+purchase and activate it.
```
Comment (dcroote) [inline docs/guides/dapis/subscribing-to-dapis/index.md:56]: This sentence reads a bit out of order given the sentence above it starts with "If a dAPI is already activated..."

File: docs/reference/dapis/understand/index.md [hunk 2]
```diff
@@ -27,102 +27,129 @@ stable coins, derivatives, NFTs, and more.
 
 ## Values stored on-chain
 
-API providers, owners of first-party Airnodes, provide the signed data used to
-store individual beacon values on-chain. A dAPI's value is in turn an aggregate
-of beacon values. dAPI values are held in the
-[Api3ServerV1.sol](https://github.com/api3dao/airnode-protocol-v1/blob/main/contracts/api3-server-v1/Api3ServerV1.sol)
-contract.
+[Datafeed values are stored on-chain](/reference/dapis/understand/#values-stored-on-chain)
```
Comment (dcroote) [inline docs/reference/dapis/understand/index.md:30]: This link is to its own section?

File: docs/reference/dapis/understand/index.md [hunk 3]
```diff
@@ -27,102 +27,129 @@ stable coins, derivatives, NFTs, and more.
...
 <img src="../assets/images/beacons.png" style="width:80%;">
 
-`Api3ServerV1.sol` manages the definitions for hundreds of dAPIs, each of which
-is an aggregated value of multiple beacons or the value of a single beacon.
+dAPIs are human-readable mappings that maps to a `beaconId` or `beaconSetId`.
+The `beaconId` for each dAPI gets updated when the price hits the set
+[deviation threshold](/reference/dapis/understand/deviations.md)/[heartbeat](/reference/dapis/understand/deviations.md#heartbeat)
+using
+[Signed-APIs](https://github.com/api3dao/signed-api/tree/main/packages/signed-api).
+[Airseeker](/reference/dapis/understand/dapis.md#airseeker) and other entities
```
Comment (dcroote) [inline docs/reference/dapis/understand/index.md:47]: 404 for Airseeker link

File: docs/reference/dapis/understand/wallets.md [hunk 4]
```diff
@@ -37,21 +36,7 @@ find a dAPI, fund its sponsor wallet, and obtain its
 > **Data feed updates:**
```
Comment (dcroote) [inline docs/reference/dapis/understand/wallets.md:36]: 404 on the `read()` link above

**Review by dcroote — CHANGES_REQUESTED** [review]
Reads very well overall 👍 

Needs update:
https://github.com/api3dao/vitepress-docs/blob/35ad682bc139b72af31768448d781c37c1f6878b/docs/guides/dapis/read-a-dapi/index.md?plain=1#L94

I found a couple of pages under [Explore](https://docs.api3.org/explore/dapis/what-are-dapis.html) that need similar updating with regards to self/managed:

404:
https://github.com/api3dao/vitepress-docs/blob/35ad682bc139b72af31768448d781c37c1f6878b/docs/explore/dapis/what-are-dapis.md#L173

This whole page:

https://github.com/api3dao/vitepress-docs/blob/35ad682bc139b72af31768448d781c37c1f6878b/docs/explore/dapis/using-dapis.md#L25-L40

File: docs/guides/dapis/subscribing-to-dapis/index.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+If a dAPI is already activated, make sure to check the expiration date and
+update parameters. You can update the parameters and extend the subscription by
+purchasing a new configuration.
+
+:::
+
+Once you've selected the dAPI and the configuration, you will be presented with
+an option to purchase the dAPI and activate it. Make sure to check the time and
+amount of the subscription. If everything looks good, click on **Purchase**.
```
Comment (dcroote) [inline docs/guides/dapis/subscribing-to-dapis/index.md:94]: This reads as a near perfect duplicate of lines 51-56 above. I think it makes more sense here.

File: docs/guides/dapis/read-a-dapi/index.md
Comment (dcroote) [inline docs/guides/dapis/read-a-dapi/index.md:1]: There is still an occurrence of `@api3/contracts/v0.8/interfaces` in lines 93-97

**Comment by vanshwassan** [comment]
> One minor change still needed, otherwise LGTM.
> 
> I might suggest merging this when the explore PR is also ready, otherwise there will be broken links and discordant content.

 I have merged `Explore_2.1_March17` into this branch and closed https://github.com/api3dao/vitepress-docs/pull/726. Also updated the links.

**Pull request merged**